sources/lib/build-system/build.dylan

```diff
@@ -69,22 +69,24 @@ end macro;
 
 
 
-// Toplevel internal function that can be invoked by Dylan Clients
-
+// Top level internal function that can be invoked by Dylan Clients
+// Arguments:
+// build-targets - A sequence of strings naming libraries to build.
```

Comment:
The strings are Jam target names such as ``clean`` or ``clean-all``  or ``exe`` or ``dll``.

Reply:
Thanks, that was rather brain dead.